src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices.Tests/Implementation/InstanceAppSITests.cs

```diff
@@ -27,13 +27,15 @@ public class InstanceAppSITests
         private readonly Mock<IOptions<PlatformSettings>> platformSettingsOptions;
         private readonly Mock<IOptionsMonitor<AppSettings>> appSettingsOptions;
         private readonly Mock<HttpMessageHandler> handlerMock;
+        private readonly Mock<HttpMessageHandler> readStatushandlerMock;
         private readonly Mock<IHttpContextAccessor> contextAccessor;
 
         public InstanceAppSITests()
         {
             platformSettingsOptions = new Mock<IOptions<PlatformSettings>>();
             appSettingsOptions = new Mock<IOptionsMonitor<AppSettings>>();
             handlerMock = new Mock<HttpMessageHandler>(MockBehavior.Strict);
+            readStatushandlerMock = new Mock<HttpMessageHandler>(MockBehavior.Loose);
```

Comment:
Why did we need a new Mock of HttpMessageHandler with MockBehavior.Loose?

Reply:
Nvm, I see the issue.

Reply:
I was having a hard time making the mock work.. but seems it runs fine with strict as well.. Updating now.